.cloudtest.yaml

```diff
@@ -47,7 +47,8 @@ executions:
   - name: "Interdomain tests packet-aws"
     env:
       - STORE_POD_LOGS_IN_FILES=true
-      - STORE_POD_LOGS_DIR=/home/circleci/project/.tests/cloud_test/$(cluster-name)
+      - STORE_POD_LOGS_DIR=/home/circleci/project/.tests/cloud_test/$(cluster-name)/logs
```

Comment:
I suppose we go away from /logs to plain?

Reply:
yep, we do not use `logs` dir on master branch.

Reply:
Fixed, thank you